frontend/src/metabase/admin/people/containers/PeopleListingApp.jsx

```diff
@@ -402,7 +402,15 @@ export default class PeopleListingApp extends Component {
     let { modal, users, groups } = this.props;
     let { error } = this.state;
 
-    users = _.values(users).sort((a, b) => b.date_joined - a.date_joined);
+    // sort the users list by last_name, ignore case or diacritical marks. If last names are the same then compare by
+    // first name
+    const compareNames = (a, b) =>
+      a.localeCompare(b, undefined, { sensitivty: "base" });
+    users = _.values(users).sort(
```

Comment:
Your change is fine, but I noticed two issues with the existing code:

1. `users` may be null, which we use to determine if we should show the loading screen. `_.values` returns `[]` for `null` which will ensure we don't call `.sort()` on `null` but overwriting `users` breaks the loading indicator
2. sorting a long list of users every time the component is rendered could be slow

It would be better to add a selector to `metabase/frontend/src/metabase/admin/people/selectors.js` like this:

```javascript:
export const getSortedUsers = createSelector(
  [getUsers],
  (users) => users && users.sort(
    (a, b) =>
      compareNames(a.last_name, b.last_name) ||
      compareNames(a.first_name, b.first_name)
  )
);
```

and use that in `PeopleListingApp`'s `mapStateToProps` instead of `getUsers`, and get rid of the reassignment of `users` in the `render` method completely.

Reply:
It was using `_.values` because `users` was a map of ID -> user

anyways I implemented the suggested changes so take a look 👀